SMPost/examples/example-normalized.jsonld

```diff
@@ -1,88 +1,175 @@
 {
   "id": "urn:ngsi-ld:SMPost:123",
   "type": "SMPost",
-  "hasPostURL": {
+    
+    "hasPostURL": 
+    {
     "type": "Property",
     "value": "http://twt.com/121"
   },
-  "postCreatedAt": {
+    
+    "postCreatedAt": {
     "type": "Property",
     "value": {
 			"@type": "DateTime",
 			"@value": "2020-12-24T12:00:00Z"
 			 }		
   },
+    
   "postId": {
     "type": "Property",
     "value": "21098319"
   },
+    
   "hasLanguage": {
     "type": "Property",
     "value": "en"
   },
+    
   "platform": {
     "type": "Property",
     "value": "Twitter"
   },
-  "hasText": {
+    
+  "hasText": [
+  {
     "type": "Property",
-    "value": "This is a tweet"
+    "value": "This is a tweet." 
   },
-  "hasImage": {
+    {
+    "type": "Property",
+    "value": "This is another tweet." 
+  }
+  ],
+    
+  "hasImages": [
+  {
     "type": "Property",
     "value": "https://twt.com/image.png"
   },
-  "hasVideo": {
+  {
+    "type": "Property",
+    "value": "https://twt.com/image2.png"
+  }],
+    
+  "hasVideos": [
+    {
     "type": "Property",
     "value": "https://twt.com/video.mp4"
   },
-  "hasPrivacy": {
+  {
+    "type": "Property",
+    "value": "https://twt.com/video2.mp4"
+  }],
+    
+  "hasPrivacyLevel": {
     "type": "Property",
     "value": "public"
   },
+    
   "location": {
     "type": "GeoProperty",
 	"value": {
-	 "type": "Point",
-	 "coordinates": 
+	"@type": "Point",

```

Comment:
type and @type are mapped in the context. no need to have @type. @coordinates should be coordinates

Reply:
Latest version is type and coordinates already, without @.

Reply:
Done.